livre/core/render/GLContext.h

```diff
@@ -33,7 +33,7 @@ namespace livre
 class GLContext
 {
 public:
-    LIVRECORE_API GLContext();
+    LIVRECORE_API GLContext( const GLEWContext* glewContext );
```

Comment:
I wont further insist on this but the idea of having a core-lib free of eq, glew, etc is slowly dying. If you want a qt output window ( no equalizer ) this is somehow enforcing you to have a glew. GLSLShaders is also having the same problem. I am not seeing in a limited time there will be alternatives to Eq but still bothers me a bit.

If you fix the styles below it will be a +1.